.rubocop.yml

```diff
@@ -31,6 +31,8 @@ Metrics/ParameterLists:
 Style/IfUnlessModifier:
   Enabled: false
 
+Style/Documentation:
+  Enabled: false
```

Comment:
Why disabling this rule? We need it in order to make sure that the API reference is updated

Reply:
done!